features/step_definitions/enrollment_summary.rb

```diff
@@ -25,3 +25,12 @@
 Given(/^the display enrollment summary configuration is enabled$/) do
   enable_feature :display_enr_summary
 end
+
+And(/^navigates to Enrollment Summary page$/) do
+  sleep(3)
```

Comment:
@vkghub is the sleep necessary? I feel like we should avoid sleeps if possible to speed in the jobs

Reply:
ok